corehq/apps/callcenter/utils.py

```diff
@@ -32,6 +32,7 @@ def valid_element_name(name):
     fields.update({
         'name': commcare_user.name or commcare_user.raw_username,
         'username': commcare_user.raw_username,
+        'user_id': commcare_user.get_id,
```

Comment:
this already gets added: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/callcenter/utils.py#L73-73